src/runtime-rs/crates/hypervisor/src/qemu/qmp.rs

```diff
@@ -713,3 +758,22 @@ pub fn get_pci_path_by_qdev_id(
     }
     None
 }
+
+/// Maximum number of SCSI devices supported
+const MAX_SCSI_DEVICES: u32 = 65535;
+
+/// Get SCSI ID and LUN from index
+fn get_scsi_id_lun(index: u32) -> Result<(u32, u32)> {
```

Comment:
What could provide an index that is bigger than MAX_SCSI_DEVICES? If we do `index: u16` we wouldn't need to do the `index > MAX_SCSI_DEVICES` if `index,MAX_SCSI_DEVICES` are `u16`.  If it is than we get truncation during runtime, I guess its better to have `u32` and check if it is out of bounds during runtime.